### DISCUSSION TITLE
Bandwidth slider

### DISCUSSION THREAD

**Discussion by sizzlik** [AlexandreRouma/SDRPlusPlus#1045] [General]
Would it be possible to make the bandwidth setting under the radio tab into a slider instead of +/- buttons (android version)? Its pretty hard to tap the edge to quick adjust by eye. Or a little visible touch point at the edges...or when the +/- keeps beeing pressed it changes faster the longer you press? Something like that would be a nice QoL addition. Typing in approx. numbers over the virtual keyboard is not much fun when hopping from freq. to freq. Dont know if im the only one constantly changing that setting, sorry if its a stupid request.
Awesome job you guys did, keep up the great work!
73

**Comment by cjcfojc** [comment]
This.  I'm running SDRPP on an S20 Ultra.  Adding that particular feature, basically anywhere on the app, would be amazing.  Frankly, changing the bandwidth really is the only thing I've struggled with (even with a stylus).  +10 for a bandwidth slider on Android.
